examples/workflows/script_loops_maps.py

```diff
@@ -0,0 +1,21 @@
+from hera.workflows import Steps, Workflow, script
+
+
+@script()
+def test_key_mapping(key_1: str, key_2: str):  # pragma: no cover
+    print(f"{key_1}:{key_2}")
```

Comment:
Until we figure out the parse/unparse problem, we need to use `format`. Other examples already do this :( 

```suggestion
    print(f"{key_1}:{key_2}")
    print("{key_1}:{key_2}".format(key_1=key_1, key_2=key_2))
```